apps/site-landing/src/containers/HomePage/landscape/Background.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import styles from './background.module.less';
+
+export const Background = () => (
+  <div className={styles.background}>
```

Comment:
now I'm looking at this again, I don't know why I put background in it's own div.

I don't want to change it now (I've spent too long on this already) but maybe future-me can try merging the background styles onto the content div